docusaurus.config.js

```diff
@@ -4,38 +4,39 @@
 // There are various equivalent ways to declare your Docusaurus config.
 // See: https://docusaurus.io/docs/api/docusaurus-config
 
-import remarkMath from 'remark-math';
-import rehypeKatex from 'rehype-katex';
+import remarkMath from "remark-math";
+import rehypeKatex from "rehype-katex";
 
-import typstRender from './src/remark/typst.js'
-import * as TypstThemes from './src/theme/typst.ts'
+import typstRender from "./src/remark/typst-render.js";
+import TypstTheme from "./src/theme/typst.ts";
 
 const config = {
-  title: 'CeTZ Documentation',
+  title: "CeTZ Documentation",
 
-  url: 'https://cetz-package.github.io/',
-  baseUrl: '/docs/',
+  url: "https://cetz-package.github.io/",
+  baseUrl: "/docs/",
 
-  organizationName: 'cetz-package',
-  projectName: 'docs',
+  organizationName: "cetz-package",
+  projectName: "docs",
 
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
 
   i18n: {
-    defaultLocale: 'en',
-    locales: ['en'],
+    defaultLocale: "en",
+    locales: ["en"],
```

Comment:
You could add [prettier](https://prettier.io/) to the repository to have a consistent style. It supports JS/TS and markdown and should respect the gitignore. `prettier -w .` to format and `prettier -c .` to check (in CI for example).
<sub>There's also [biome](https://biomejs.dev/), but it doesn't support markdown right now.</sub>

Reply:
I've added it thanks!